src/utils.js

```diff
@@ -1215,3 +1215,43 @@ export function mergeDeep(target, ...sources) {
 
   return mergeDeep(target, ...sources);
 }
+
+/**
+ * returns a hash of a string using a fast algorithm
+ * source: https://stackoverflow.com/a/52171480/845390
+ * @param str
+ * @param seed (optional)
+ * @returns {string}
+ */
+export function simpleHash(str, seed = 0) {
```

Comment:
any chancce you can call this cyrb53Hash or something describing what it is; the source you chose has many hashes.